torch/autograd/__init__.py

```diff
@@ -305,6 +305,8 @@ def variable(*args, **kwargs):
 from torch._C._autograd import (DeviceType, ProfilerActivity, ProfilerState, ProfilerConfig, ProfilerEvent,
                                 _enable_profiler_legacy, _disable_profiler_legacy, _profiler_enabled,
                                 _enable_record_function, _set_empty_test_observer, kineto_available,
+                                register_execution_graph_observer_once, enable_execution_graph_observer,
```

Comment:
Do we really want to expose these as public APIs in `torch.autograd`? Shouldn't they be in the `torch.profiler`?

Reply:
Thanks @albanD, this diff is still WIP. I need to do a few more passes before it's ready for review. Will ping you when it's ready :).

Reply:
@albanD, the code is mostly ready for review. There were some windows build issues, I'm trying to resolve. Otherwise, it's ready for review.